src/main/java/com/money_account_service/utility/UserServiceClient.java

```diff
@@ -0,0 +1,44 @@
+package com.money_account_service.utility;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.money_account_service.dtos.response.AuthorizeResponseDto;
+import lombok.AllArgsConstructor;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+
+@AllArgsConstructor
+public class UserServiceClient {
+
+    private OkHttpClient okHttpClient;
+    public static final String AUTHORIZE_URL = "https://localhost:8081/authorize";
+
+    public AuthorizeResponseDto authorize(String accessToken) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
It's better to inject `ObjectMapper`. It should be a bean. This object is quite expensive to create.